modules/reference/pages/command/featureUtility-installFeature.adoc

```diff
@@ -33,6 +33,12 @@ Install the MicroProfile Health 2.2 feature without caching the feature to the l
 featureUtility installFeature mpHealth-2.2 --noCache
 ----
 
+Install the userFeature-1.0 feature from a Maven repository:
```

Comment:
For clarity:

Install the `userFeature-1.0` custom user feature from a Maven repository: